compiler/ic/ic.nim

```diff
@@ -1126,7 +1148,7 @@ proc rodViewer*(rodfile: AbsoluteFile; config: ConfigRef, cache: IdentCache) =
   let err = loadRodFile(rodfile, m, config, ignoreConfig=true)
   if err != ok:
     echo "Error: could not load: ", rodfile.string, " reason: ", err
-    quit 1
+    doAssert false # quit would prevent getting a stacktrace
```

Comment:
Users shouldn't see stack traces, it's sloppy code. Number of times I got a stacktrace from GCC for an internal error: 0.

Reply:
reverted to `quit 1` for now, but maybe future work can still use doAssert for `-d:nimDebug`, which makes sense (there's -d:debug but `-d:nimDebug` would be passable without recompiling nim, via using `isDefined`)

Reply:
=> https://github.com/nim-lang/Nim/pull/17739